### ISSUE TITLE
Milestone - Add support for all queries available in the data portal available via the public API (4.8)

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/issues#854]

**Comment by ssarrafan** [comment]
Discussed today.  Some of this work is complete. May have additional updates based on user research (#857). Keeping open for now.  
FYI @aclum @ljohnson09

**Comment by aclum** [comment]
no, the plan is to identify high priority actions from the user research to address this milesone. cc @shreddd  @eecavanna

**Comment by ssarrafan** [comment]
This milestone was marked completed on https://docs.google.com/spreadsheets/d/1vjBvnMO2ELg7-pFNnrJn546DhRih9PSwoF9hCz0ksxo/edit?gid=948982184#gid=948982184

Complex queries via the queries:run endpoint enable replication of data portal queries programmatically. A mechanism for an endpoint supporting a RDF graph representation is being developed via graph API for nmdc-schema metadata; this will require less knowledge of the NMDC schema and be more flexible as the schema evolves.

@aclum can we close this?

**Comment by aclum** [comment]
The comment above pertains to work done in a different year.  This is a rolling milestone. @eecavanna  and @shreddd  have identified items to work on this quarter. Please link to them here @shreddd

**Comment by ssarrafan** [comment]
Per @shreddd  this is done now. Donny just checked in traversal endpoint which means we can do these queries now.  Sujay has rolled up study views.  We have functional ways to pull data portal info from the API.  
Closing this one.  FYI @aclum
